API/src/routes/Assessment/index.js

```diff
@@ -9,11 +9,9 @@ module.exports = server => {
     `${BASE_URL}/submit`,
     async (req, res, next) => {
       try {
-        const { assessment } = req.params;
-
-        // verify that your data is making it here to the API by using console.log(assessment);
-        // call the AssessmentService.submit function from the API/src/microservices/Assessment/ and
-        // supply the correct parameters
+        const { formatted_assessment } = req.params;
+        console.log(formatted_assessment);
+        AssessmentService.submit(formatted_assessment);
```

Comment:
```suggestion
        const new_assessment = AssessmentService.submit(formatted_assessment);
```
This is how you'd return the database's assessment object from a function call. Then below you could pass it into the response handler the same way the get request does on this same file.